### PR TITLE
Add kvm dashboards the home KVM home page

### PR DESCRIPTION
**Pull request by QuentinBisson**
Closes https://github.com/giantswarm/giantswarm/issues/18217

This PR

- adds KVM dashboard to the Home page on KVM

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Gauss:
image

Geckon:
image

**Review by marians — APPROVED** [review]
I don't understand how this works, but the outcome looks good to me!

File: helm/dashboards/templates/configmap.yaml
Comment (QuentinBisson) [inline helm/dashboards/templates/configmap.yaml:9]: @marians this line makes ensure that helm templates the `home.json` dashboard instead of just rendering it as is so we can use helm values in there :)

**Comment by QuentinBisson** [comment]
I am waiting for an approval from @giantswarm/team-atlas as well for the technical approach

**Pull request merged**
